### ISSUE TITLE
pandoc dependency

### ISSUE THREAD

**Issue by ax3l** [openPMD/openPMD-viewer#152]
### Pandoc

I upgraded a Python 2.7 install of openPMD-viewer `0.3.3` to `0.5.3` via pip today (note: there is still a tag for `0.5.3` missing on GitHub -> Releases) and noted the following install problem:

```
$ pip install openPMD-viewer --user -U
[....]

Collecting openPMD-viewer
  Using cached openPMD-viewer-0.5.3.tar.gz
    Complete output from command python setup.py egg_info:
    /home/axel/.local/lib/python2.7/site-packages/pkg_resources/__init__.py:1869: UserWarning: /usr/lib/pymodules/python2.7/rpl-1.5.5.egg-info could not be properly decoded in UTF-8
      warnings.warn(msg)
    Maybe try:
    
        sudo apt-get install pandoc
    See http://johnmacfarlane.net/pandoc/installing.html
    for installation options
```

`pandoc` is missing in the (pip) `requirements.txt` and besides the python part it also needs the printed binary-part to be installed. Anyway, it's probably only used for documentation building or rst file creation for packaging?

Not sure when this issue was introduced, because it's in `setup.py` already for a long time. Note that I installed previous versions (`0.3.3`) via `python setup.py install` from the sources directly, that might be the reason it was not triggered.

### Python 3.4

Repeating the same install via Python 3.4 seemed fine even without pandoc, but only installs `0.5.2` o.0

Also, are we sure we drop support and CI for Python 3.4 already (see setup.py meta information)? It's still the stable Python release in many distributions.

**Comment by RemiLehe** [comment]
Thanks for reporting these issues.

Regarding `pandoc` and `pypandoc`: they are not a requirement of openPMD-viewer. 
- If `pypandoc` is not installed on your system, `openPMD-viewer` will detect it and will not try using it (it's only use is when we try to push to PyPI, as it only supports `rst`).
- If `pypandoc` is **properly installed**, `openPMD-viewer` will use it to convert the README.
The problem is when `pypandoc` is not properly installed, i.e. when the Python wrapper is installed but not the `pandoc` backend. This typically happens when installing `pypandoc` with `pip` instead of `conda`.

So, in summary, the issue you observe is not really a problem of `openPMD-viewer` but rather a problem of your installation of `pypandoc`. You can try `pip uninstall pypandoc` or conversely `sudo apt-get install pandoc` (in order to install the backend).

Regarding the Python 3.4 issue: I am drafting a new PyPI release right now, which will support Python 3.4.

**Comment by RemiLehe** [comment]
Done: https://pypi.python.org/pypi/openPMD-viewer/0.5.4

**Comment by ax3l** [comment]
> This typically happens when installing pypandoc with pip instead of conda.

very good point, that's the issue. Thank you!
